.github/workflows/sycl-linux-build.yml

```diff
@@ -58,9 +58,13 @@ on:
         type: string
         default: llvm_sycl.tar.zst
 
-      pack_release:
+      release_toolchain_artifact:
         type: string
         required: false
+        description: |
+          If provided, create an addition toolchain artifact with the same name
+          (`inputs.toolchain_artifact_filename`) as `toolchain_artifact` above
```

Comment:
This is a bit confusing. It's an archive name, not an artifact name, isn't it? From this sentence it seems like it creates one more "sycl_linux_default" or so. Can't we say "If provided, create an additional toolchain artifact without utilities used for testing called `inputs.release_toolchain_artifact` or just "If provided, create an additional toolchain artifact without utilities used for testing"?